src/mlpack/methods/grad_boosting/grad_boosting.hpp

```diff
@@ -0,0 +1,237 @@
+/**
+ * @file methods/grad_boosting/grad_boosting.hpp
+ * @author Abhimanyu Dayal
+ *
+ * Gradient Boosting class. 
+ *
```

Comment:
Please add more explanation of what is Gradient Boost, the article, and a Bibtex reference for it.